specs/rm/requirements-management-shapes.ttl

```diff
@@ -172,28 +170,17 @@ or to satisfy a contract, standard, specification, or other formally imposed doc
     oslc:propertyDefinition oslc:instanceShape  ;
     oslc:range oslc:ResourceShape ;
     oslc:occurs oslc:Zero-or-one;
-    dcterms:description "Resource Shape that provides hints as to resource property value-types and allowed values"^^rdf:XMLLiteral ;
+    dcterms:description "Resource Shape that provides hints as to resource property value-types and allowed values."^^rdf:XMLLiteral ;
     oslc:valueType oslc:Resource ;
     oslc:representation oslc:Reference .
 
 <#shortTitle> a oslc:Property ;
     oslc:name "shortTitle" ;
     oslc:occurs oslc:Zero-or-one;
-    dcterms:description "Short name identifying a resource, often used as an abbreviated identifier for presentation to end-users. SHOULD include only content that is valid inside an XHTML &lt;span&gt element."^^rdf:XMLLiteral ;
+    dcterms:description "Short name identifying a resource, often used as an abbreviated identifier for presentation to end-users. SHOULD include only content that is valid inside an XHTML &lt;span&gt; element."^^rdf:XMLLiteral ;
     oslc:propertyDefinition  oslc:shortTitle ;
     oslc:valueType rdf:XMLLiteral .
 
-<#modifiedBy> a oslc:Property ;
```

Comment:
was it used anywhere? fine by me if not, this is not an RDF property.

Reply:
No, it was not used.